docs/Proficiency Exams/CS173.md

```diff
@@ -0,0 +1,39 @@
+# CS173
+
+[CS173 (Discrete Structures)](../Course%20Wiki/CS%20Course%20Offerings/CS173.md) is a theory focused CS course that is required for Computer Engineering and Computer Science majors. It also counts as a technical elective for Electrical Engineering majors.
+
+## Who Should Take the Exam
+
+For students who want to take [CS225](../Course%20Wiki/CS%20Course%20Offerings/CS225.md) early, they are required to take CS173 before taking CS225. Since Computer Engineering students are required to take both [ECE120](../Course%20Wiki/ECE%20Course%20Offerings/ECE120.md) and [ECE220](../Course%20Wiki/ECE%20Course%20Offerings/ECE220.md) before taking CS173, they will usually take CS225 one semester later than Computer Science Students. If you want to take CS225 as soon as possible, take the CS173 proficiency exam. [MATH213](../Course%20Wiki/MATH%20Course%20Offerings/MATH213.md) is an alternative for this class that doesn't require ECE220. However, MATH213 is considered to be disorganized compared to CS173. MATH213 also doesn't include some topics that will be used in [CS374](../Course%20Wiki/CS%20Course%20Offerings/CS374.md)
+
+It's not an easy proficiency exam. However, like all proficiency exams, there is no penalty for failing the CS173 proficiency exam, so if you're on the fence, take the exam.
+
+## Content Covered
+
+- Logic
+- Number Thoery
+- Modular Arithmetic and Sets
+- Relations
+- Functions
+- Graphs
+- Two-way bounding
```

Comment:
is this just finding bounds on functions?

Reply:
It covers finding bounds on other topics as well